src/components/campaigns/Summary/Goal/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import numbro from 'numbro'
+import Stat from '../Stat'
+
+const Goal = ({ loading, goal, currencySymbol, numberFormat }) => {
+  if (!goal) return <span />
+  const formattedGoal = numbro(goal).format(currencySymbol + numberFormat)
+
+  return (
+    <Stat
+      loading={loading}
+      number={formattedGoal} />
+  )
+}
+
+Goal.displayName = 'CampaignSummaryGoal'
+
+Goal.propTypes = {
+  amount: React.PropTypes.string,
+  currencySymbol: React.PropTypes.string,
+  numberFormat: React.PropTypes.string,
+  loading: React.PropTypes.bool
+}
```

Comment:
Should these propTypes be required? I know we get defaults from the parent, but this component shouldn't assume that knowledge.